### DISCUSSION TITLE
Random balloon/hat color

### DISCUSSION THREAD

**Discussion by PhyxionNL** [OpenRCT2/OpenRCT2#15647] [Feature requests] [answered]
Currently all balloons/hats are the same color, it would be nice to add an option to give the customer a random balloon/hat color.

**Comment by pizza2004** [comment]
Also umbrellas!

**Comment by pizza2004** [comment]
Ooh, or an options to let peeps buy the same color as their shirt!

**Comment by PhyxionNL** [comment]
Just all items users can buy :+1:

**Comment by Gymnasiast** [comment]
I have reopened this as I think it would probably be feasible to implement this in a non-intrusive way. I can't give guarantees it will actually be implemented, though.

**Comment by PhyxionNL** [comment]
Just a single checkbox like : **Random color** should be fine. There's plenty of space on the options page.

**Comment by say25** [comment]
Semi-related this issue https://github.com/OpenRCT2/OpenRCT2/issues/6549

**Comment by Gymnasiast** [comment, marked as answer]
This feature has been implemented.
